src/account.js

```diff
@@ -0,0 +1,23 @@
+
+/* Account login state manager
+ */
+
+import _ from 'lodash'
+import db from './db'
+import { storeKey } from './config'
+
+export default { setUser }
```

Comment:
Yeah, I think this is also a preference thing. Exporting functions directly in ES6 has caused us a ton of pain as far as testing goes, so in this and the `server.js` file I approached it from the "export a single object that contains your functions" approach that was used a lot in ES5, which is more testable. It also is more clear name-wise, since we can enforce `setUser` as a function name rather than leaving it up to the requiring code to decide how to name the function that is being exported.

In the interest of consistency though I'll change it to just export the single function as a default.